tests/test_unary_pointwise_ops.py

```diff
@@ -303,26 +303,44 @@ def test_accuracy_isfinite(shape, dtype):
     gems_assert_equal(res_out, ref_out)
 
 
-@pytest.mark.parametrize("shape", DIM_POINTWISE_SHAPES)
-@pytest.mark.parametrize("dtype", ALL_FLOAT_DTYPES + ALL_INT_DTYPES)
-@pytest.mark.parametrize("dims", DIMS)
-def test_accuracy_flip(shape, dtype, dims):
+def get_max_ndim(shape, dims):
+    max_ndim = max(len(shape), len(dims))
+    for dim in dims:
+        dim = abs(dim) + 1
+        if dim > max_ndim:
+            max_ndim = dim
+    return max_ndim
```

Comment:
Is this used to make a tensor have at least N dimensions, such that each axis in dims is a valid axis in the tensor?

Then if axis is not negative, the ndim is required to be `ndim >= axis + 1`; and when the axis not positive, the ndim is requires to be `ndim >= abs(axis)`.



For example, a tensor is only requires to have at least 3d to get its axis `-3`.

```text
(0, 1, 2, 3)
(-4, -3, -2, -1)
```

Reply:
> Is this used to make a tensor have at least N dimensions, such that each axis in dims is a valid axis in the tensor?
> 
> Then if axis is not negative, the ndim is required to be `ndim >= axis + 1`; and when the axis not positive, the ndim is requires to be `ndim >= abs(axis)`.
> 
> For example, a tensor is only requires to have at least 3d to get its axis `-3`.
> 
> ```
> (0, 1, 2, 3)
> (-4, -3, -2, -1)
> ```

Done.